### PR TITLE
Implement javascript package.

### PR DESCRIPTION
**Pull request by mattcollier**

### CODE REVIEW

**Comment by mattcollier** [comment]
I would like to understand our release plan for this and other context modules.  The pattern I've been using elsewhere is that every change is a major release.  So, 1.0.0 will be released immediately after this is merged and we go from there, is that right?

**Comment by davidlehn** [comment]
This is putting js code and packaging in the gh-pages branch.  Is that what we want?  It's a w3c repo, so should probably also have appropriate license file.  Also should consider a README (as was suggested in #3) that explains the situation.

**Comment by mattcollier** [comment]
I think yes on using the gh-pages branch (for now anyway).  Alternatively, one can tell ghpages to use any branch right?  So, if putting all this on master makes more sense, I'm fine with that.

I'm not interested in writing a README.  Anyone else up for the task?

What is the appropriate license?  BSD-3?

**Comment by davidlehn** [comment]
I'm not saying just use a different name.  I was more wondering about wisdom of serving up non-web things via gh-pages.  But using multiple branches means we'd have to keep the context in sync, which would be a pain.

No idea what licenses we can/should use here.  @msporny do you know?

**Review by dmitrizagidulin — APPROVED** [review]
👍 !

**Comment by msporny** [comment]
> No idea what licenses we can/should use here. @msporny do you know?

We have to use the W3C documentation/code license... don't have time to go looking for it, but it's on W3C's site somewhere (and possibly linked to from the VC spec).

**Comment by mattcollier** [comment]
@davidlehn exactly, the sync issue is what we're trying to avoid, and I looked for github symlink magic... but such magic does not exist apparently.  There's some submodule thing which I don't understand which may have some implications here, idk.

**Comment by dmitrizagidulin** [comment]
@davidlehn Don't worry about the gh-pages aspect. It doesn't really matter. Our Node libs will just use npm (they don't care about branches), and the spec will be served off of gh-pages as usual.

**Comment by dmitrizagidulin** [comment]
Also, I think we should merge this PR and add a separate issue to add a license file. No sense in having that hold this up.

**Comment by mattcollier** [comment]
I've add a license file based on this example: https://github.com/w3c/publishing/blob/master/LICENSE.md

**Comment by mattcollier** [comment]
@dlongley @davidlehn @msporny need an approval here.

**Review by dlongley — APPROVED** [review]

File: package.json
Comment (davidlehn) [inline package.json:5]: `npm pack` is pulling in all the website stuff.  Only need to ship the `js` and `contexts` contents:
```suggestion
  "main": "js",
  "files": [
    "contexts",
    "js"
  ],
```

**Pull request merged**
